File: trunk/client/python/testremove.py

import sphivedbcli
import time
import sys

if __name__ == "__main__":
	if len( sys.argv ) != 2:
		print "Usage: %s <config file>" % ( sys.argv[0] )
		print "\tpython %s ../../sphivedb/sphivedbcli.ini" % ( sys.argv[0] )
		sys.exit( -1 )

	configFile = sys.argv[1]

	cli = sphivedbcli.SPHiveDBClient()

	cli.init( configFile )

	try:
		resp = cli.remove( 0, "foobar", "addrbook" )

		print resp

	except Exception, e:
		print e

File: trunk/client/php/testremove.php
<?php
/*
 * Copyright 2009 Stephen Liu
 * For license terms, see the file COPYING along with this library.
 */

include_once "sphivedbcli.php";

if( $argc != 2 ) {
	printf( "Usage: %s <config file>\n", $argv[0] );
	printf( "\tphp %s ../../sphivedb/sphivedbcli.ini\n", $argv[0] );
	return -1;
}

$configFile = $argv[1];

$client = new SPHiveDBClient();

$ret = $client->init( $configFile );

$resp = $client->remove( 0, "foobar", "addrbook" );

print $resp

?>

